guardrails/schema.py

```diff
@@ -226,7 +227,7 @@ def get_validators(self, strict: bool = False) -> List[Validator]:
                 # beginning of a rail file.
 
             # Create the validator.
-            _validators.append(validator(*args, on_fail=on_fail))
+            _validators.append(validator(document_store, *args, on_fail=on_fail))
```

Comment:
This is causing issues with validators that don't use a document store.  Since the function arguments are passed as a spread the order needs to be maintained.  This means that either all validators need to add a `document_store` arg at the beginning of their method signature, or the document_store on the base class Validator needs to become optional so this can move to the end of the signature and be passes as a kwarg instead.  I think the first option is the better one.